04_setup_ironic.sh

```diff
@@ -78,3 +78,12 @@ sudo podman run -d --net host --privileged --name ironic --pod ironic-pod \
 # Start Ironic Inspector 
 sudo podman run -d --net host --privileged --name ironic-inspector \
      --pod ironic-pod -v $IRONIC_DATA_DIR:/shared "${IRONIC_INSPECTOR_IMAGE}"
+
+sudo mkdir -p /etc/openstack
+sudo cat > /etc/openstack/clouds.yaml <<EOF
```

Comment:
This is going to replace any existing file or settings that might already be present. I think we should generate the file to ./ocs like we do with the kube auth file and suggest that the user copy it (or its contents) to their home directory.